.github/workflows/main.yaml

```diff
@@ -16,24 +16,21 @@ jobs:
         with:
           fetch-depth: 1
       - name: Setup Ruby
-        uses: ruby/setup-ruby@55283cc23133118229fd3f97f9336ee23a179fcf # v1.146.0
+        uses: ruby/setup-ruby@97e35c5302afcf3f5ac1df3fca9343d32536b286 # v1.184.0
         with:
-          ruby-version: '3.1' # Not needed with a .ruby-version file
+          ruby-version: '.ruby-version' # Not needed with a .ruby-version file
```

Comment:
I think more tools may be looking at .ruby-version (at least renovate noted it for me) and it may in general be easier to remember to update it than monitoring this file with a lot of version numbers.